automated install/basic-install.sh

```diff
@@ -764,6 +764,7 @@ setDNS() {
       Norton ""
       Comodo ""
       DNSWatch ""
+      Quad9 ""
       Custom "")
   # In a whiptail dialog, show the options
   DNSchoices=$(whiptail --separate-output --menu "Select Upstream DNS Provider. To use your own, select Custom." ${r} ${c} 6 \
```

Comment:
You should increment the `6` value so all options are made sure to be shown at once.